upgrading/topics/rhsso/intro.adoc

```diff
@@ -41,3 +43,20 @@ installations can also rollback the patch and server configuration.
 A micro release only contains the artifacts that have changed. For example if Red Hat Single Sign-On 7.2.1 contains changes to
 the server and the JavaScript adapter, but not the EAP adapter, only the server and JavaScript adapter are released and require
 updating.
+
+=== Migration from Keycloak
+
+You can migrate to Red Hat Single Sign-On, the supported Red Hat product, from Keycloak, the community project. Some differences exist in the installed software, such as the following:
+
+* Red Hat Single Sign-On uses JBoss EAP in place of WildFly for the built-in application server
+* Some Keycloak features are not included in Red Hat Single Sign-On, such as some of the Service Provider Interfaces (SPIs).
```

Comment:
Remove this. This is not correct. SPIs that are supported in community are also supported in product. SPIs that are not supported are available in both community and product.

Reply:
Okay, I have removed it.  However, I know we only document the User Storage SPI in SSO.  Is the difference that do not document all SPIs in SSO?